app/Models/BeatmapsetArchive.php

```diff
@@ -5,13 +5,18 @@
 
 namespace App\Models;
 
+use App\Exceptions\BeatmapProcessorException;
+
 class BeatmapsetArchive
 {
     public function __construct(string $osz)
     {
         $this->osz = $osz;
         $this->zip = new \ZipArchive();
-        $this->zip->open($this->osz);
+        $code = $this->zip->open($this->osz);
```

Comment:
not actually relevant to this pr but this class can use some property declarations